app/controllers/api/projects_controller.rb

```diff
@@ -74,7 +74,8 @@ def find_project_as_json_with_dependencies!(platform, name, version_name, subset
                 raise ActionController::BadRequest.new("Unsupported subset")
               end
 
-    project = Project.find_with_includes!(platform, name, includes)
+    project = Project.find_best(platform, name, includes)
+    raise ActiveRecord::RecordNotFound unless project
```

Comment:
Why not just use `find_best!` here instead of raising on `nil`

Reply:
Yep! Just swapped it!